### DISCUSSION TITLE
How hide some field in modal when create new row?

### DISCUSSION THREAD

**Discussion by IvanovRoman** [KevinVandy/mantine-react-table#182] [Q&A] [answered]
When adding a new row in table, I don't have to enter data in column `Created` and `Updated`, because filled out automatically. In modal must show only three columns: `Name, Alias, Description`.
![image](https://github.com/KevinVandy/mantine-react-table/assets/13165973/7ce5d886-76fa-487f-8096-4cf943ec0b42)

**Comment by KevinVandy** [comment]
You can just put `Edit: null` in the column definition

**Reply by IvanovRoman** [reply, marked as answer]
Maybe? `Edit: () => null`
I write
```
{
      accessorKey: 'updatedAt',
      header: 'Updated',
      enableEditing: false,
      Edit: null
}
```
Typescript write error:
`Type 'null' is not assignable to type '((props: { cell: MRT_Cell<Category>; column: MRT_Column<Category>; row: MRT_Row<Category>; table: MRT_TableInstance<Category>; }) => ReactNode) | undefined'.`

**Comment by IvanovRoman** [comment]
Another way to hide a field. 
Add this field:
```
mantineEditTextInputProps: {
        sx: {
          display: 'none'
        }
      },
```
Example:
```
const columns: MRT_ColumnDef<any>[] = [
{
      accessorKey: 'updatedAt',
      header: 'Updated',
      mantineEditTextInputProps: {
        sx: {
          display: 'none'
        }
      },
},
];
```
